packages/instant/src/components/instant_heading.tsx

```diff
@@ -47,11 +73,11 @@ export const InstantHeading: React.StatelessComponent<InstantHeadingProps> = pro
             <Flex direction="column" justify="space-between">
                 <Container marginBottom="5px">
                     <Text fontSize="16px" fontColor={ColorOption.white} fontWeight={500}>
-                        {displaytotalEthBaseAmount(props)}
+                        {loadingOrAmount(props.quoteState, displaytotalEthBaseAmount(props))}
```

Comment:
Feels kind of weird to have two nested `Text` components. Can't `displayTotalEthUsdAmount` take care of all of this?

Reply:
Good call -- yes, will fix